BTW, I got the error when installing:

```
npm ERR! peerinvalid The package react@0.14.7 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-binding-component@1.0.3 wants react@0.14.3
npm ERR! peerinvalid Peer react-dom@0.14.7 wants react@^0.14.7
```

May I ask you to fix this as well?

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "weflex-ui",
-  "version": "2.7.3",
+  "version": "2.7.4",
   "description": "The UI Component that providing basic form controls which is compatible with React.js",
   "main": "lib/index.js",
   "scripts": {
@@ -37,7 +37,7 @@
     "rc-select": "5.10.4",
     "rc-upload": "1.9.0",
     "rc-util": "3.1.3",
-    "react-binding-component": "1.0.3",
+    "react-binding-component": "1.0.5",
     "warning": "2.1.0"
   },
   "devDependencies": {

